rpm/generic/zfs.spec.in

```diff
@@ -522,7 +522,7 @@ systemctl --system daemon-reload >/dev/null || true
 %config(noreplace) %{_sysconfdir}/%{name}/vdev_id.conf.*.example
 %attr(440, root, root) %config(noreplace) %{_sysconfdir}/sudoers.d/*
 
-%config(noreplace) %{_sysconfdir}/bash_completion.d/zfs
+%config(noreplace) /usr/share/bash-completion/completions
```

Comment:
I think this should probably just be:

```
%{_datadir}/bash-completion/completions
```

(`%{_datadir}` expands to `/usr/share` and since the files live in `/usr` now, `%config(noreplace)` should probably be removed so RPM doesn't try to preserve edits.)

Reply:
This would break building on RHEL and related systems, which do expect bash_completion files in /etc/bash_completion.d